compiler/quantization-value-test/CMakeLists.txt

```diff
@@ -0,0 +1,136 @@
+  set(RECIPE_SOURCE_FILE "${TEST}.recipe")
+  set(RECIPE_OUTPUT_FILE "${TEST}.tflite")
+  set(CIRCLE_OUTPUT_FILE "${TEST}.circle")
+
+  # Copy .recipe
+  add_custom_command(OUTPUT "${RECIPE_SOURCE_FILE}"
+                     COMMAND ${CMAKE_COMMAND} -E copy_if_different
+                             "${TEST_REPO}/${RECIPE}" "${RECIPE_SOURCE_FILE}"
+                     DEPENDS "${TEST_REPO}/${RECIPE}"
+                     COMMENT "Generating ${RECIPE_SOURCE_FILE}")
+
+  # Generate .tflite
+  add_custom_command(OUTPUT "${RECIPE_OUTPUT_FILE}"
+                     COMMAND tflchef-file "${RECIPE_SOURCE_FILE}" "${RECIPE_OUTPUT_FILE}"
+                     DEPENDS tflchef-file "${RECIPE_SOURCE_FILE}"
+                     COMMENT "Generating ${RECIPE_OUTPUT_FILE}")
+
+  # Generate .circle
+  add_custom_command(OUTPUT "${CIRCLE_OUTPUT_FILE}"
+                     COMMAND tflite2circle "${RECIPE_OUTPUT_FILE}" "${CIRCLE_OUTPUT_FILE}"
+                     DEPENDS tflite2circle "${RECIPE_OUTPUT_FILE}"
+                     COMMENT "Generating ${CIRCLE_OUTPUT_FILE}")
+
+  list(APPEND TEST_DEPS "${CIRCLE_OUTPUT_FILE}")
+endforeach(TEST IN ITEMS ${QUANTIZATION_VALUE_TEST})
+
+###
+### Generate test.config
+###
+set(TEST_CONFIG "${CMAKE_CURRENT_BINARY_DIR}/test.config")
+
+add_custom_command(
+  OUTPUT ${TEST_CONFIG}
+  COMMAND ${CMAKE_COMMAND} -E remove -f ${TEST_CONFIG}
+  COMMAND ${CMAKE_COMMAND} -E echo 'RECORD_MINMAX_PATH=\"$<TARGET_FILE:record-minmax>\"' >> ${TEST_CONFIG}
+  COMMAND ${CMAKE_COMMAND} -E echo 'CIRCLE2CIRCLE_PATH=\"$<TARGET_FILE:circle2circle>\"' >> ${TEST_CONFIG}
+  COMMAND ${CMAKE_COMMAND} -E echo 'CIRCLE_TENSORDUMP_PATH=\"$<TARGET_FILE:circle-tensordump>\"' >> ${TEST_CONFIG}
+  DEPENDS record-minmax
+  DEPENDS circle2circle
+  DEPENDS circle-tensordump
+  COMMENT "Generate test configuration"
+)
+
+list(APPEND TEST_DEPS "${TEST_CONFIG}")
+
+# This enforces CMake to generate all the dependencies during "build" phase
+add_custom_target(quantization_value_test_deps ALL DEPENDS ${TEST_DEPS} quantization-value-test_python_deps)
+
+# Run tests
+add_test(
+  NAME fake_wquant_value_test
+  COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/test_fake_wquant.sh"
+          "${TEST_CONFIG}"
+          "${CMAKE_CURRENT_BINARY_DIR}"
+          ${QUANTIZATION_VALUE_TEST_WITH_PARAM}
+)
+
+add_test(
+  NAME record_minmax_value_test
+  COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/test_record_minmax.sh"
+          "${TEST_CONFIG}"
+          "${CMAKE_CURRENT_BINARY_DIR}"
+          ${QUANTIZATION_VALUE_TEST_WITH_PARAM}
+)
+
+add_test(
+  NAME quantization_value_test
+  COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/test_quantization.sh"
+          "${TEST_CONFIG}"
+          "${CMAKE_CURRENT_BINARY_DIR}"
+          ${QUANTIZATION_VALUE_TEST_WITH_PARAM}
+)
+
+set_tests_properties(record_minmax_value_test PROPERTIES DEPENDS fake_quantization_value_test)
+set_tests_properties(quantization_value_test PROPERTIES DEPENDS record_minmax_value_test)
```

Comment:
Code itself looks good to me... but the name seems to be too general ``). Unfortunately, I can't suggest any better name T_T

Reply:
> Code itself looks good to me... but the name seems to be too general ``)

Our quantization scheme can be described as post-training affine quantization. But post-training-affine-quantization-value-test is too long, so how about using `paq-value-test` as the directory name? Each test names will be `paq_fake_quant_test`, `paq_record_minmax_test`, `paq_quantization_test`

 @seanshpark What do you think of this?

Reply:
pronouncing is somewhat strange with `paq`... as word is `post-training-affine`, I could see `pta`.
- `pta_fake_quant_test`, `pta_record_minmax_test`, `pta_quantization_test`

Reply:
I thought `pta` before, but it is the abbreviation of "please try again".
(https://en.dict.naver.com/#/entry/enko/a0aeb300a9704137baa73fff43df1c47)

I think "please try again" quantization sounds that our tool does not work well..

Reply:
Just `pa-` ?

Reply:
> Just pa- ?

Ok. I will go with `pa-quantization-value-test`. Cmake test names will be `pa_fake_quant_test`, `pa_record_minmax_test`, `pa_quantization_test`.